### DISCUSSION TITLE
Does my Hubs Cloud region have to match my AWS region?

### DISCUSSION THREAD

**Discussion by BethanyWinslow** [mozilla/hubs#3338] [Hubs Cloud Questions] [answered]
I didn't see anyone with this same question, but when I created my AWS account it had automatically assigned me Ohio so that's what I used to create my SSH Keypair. I'm subscribing to Hubs Cloud and Robin's very helpful,(thank you Robin for your videos!!) video How to Deploy Hubs Cloud to AWS: Part 2 says I should choose the region closest to me which is actually California. It was very clear that the SSH keypair has to match, but should I choose Ohio just to keep everything the same? I'm hoping I can change this later if I need to. Anybody's help is most appreciated. Thank you Mozilla Hubs developers!

**Comment by robinkwilson** [comment, marked as answer]
I'm so glad the videos are helping you! 😄 

The ssh keypair needs to be in the same region as your stack deploy so keep using Ohio and choose Ohio as your stack deploy region!

Here's the available Hubs Cloud regions (Ohio is one of them):

![image](https://user-images.githubusercontent.com/29560735/99590969-f530f800-29a2-11eb-91fa-af1598a27377.png)

**Reply by BethanyWinslow** [reply]
Thanks Robin! When I went through the process a few days ago, I ended up not seeing the keypairs at all, (I think because I changed to AWS to Oregon). So I went through the process again and set up the keypairs and region to Oregon. Looks like the stack is working fine. So far so good! Thanks again!
